### ISSUE TITLE
Bug: Search bar style is ignored when "Clock - Fill screen height" is unset

### ISSUE THREAD

**Issue by maximbaz** [MM2-0/Kvaesitso#350]
Start from fresh installation. 
Observe the search bar background is transparent. 
Open settings, unset "Clock - Fill screen height" 
Observe that the search bar is no longer transparent. 
Try to change search bar style in settings and observe that they don't have any effect. 

version 1.24.1
de.mm20.launcher2.release
versionCode 2023041800

**Comment by exstntlmsnthrp** [comment]
You are correct. I just noticed that on my existing installation when I made the change. Here are a couple of screenshots to demonstrate. 

Before:

GitHub is having issues with uploads. I will edit and upload later.


After:
